### DISCUSSION TITLE
Default colorscheme for all users

### DISCUSSION THREAD

**Discussion by NonSwag** [VOUNTY/Wizard#6] [Ideas]
An option which defines the default style of the website (either dark or light mode)
This option will define the style of the website when no application data was found for the `theme` key
I am terrible at explaining, so feel free to ask if something is unclear.

**Comment by GradientTim** [comment]
The color scheme is automatically adapted to the end consumer. 
This means that when a consumer uses the web interface, the theme is used by their operating system.

Therefore, it will remain as it is for now, but may change again in the future.
Alternatively, the primary and secondary colors of the web interface can be changed.

- [Overview in the WizardWeb](https://github.com/VountyNetwork/WizardWeb/blob/master/src/index.tsx#L21)
- [Mantine useColorScheme() hook](https://mantine.dev/hooks/use-color-scheme/)
- [Change primary and secondary color](https://docs.vounty.net/wizard/admin/configs/#website)
